src/parsley/ui.js

```diff
@@ -170,16 +170,28 @@ UI.Field = {
               .addClass('parsley-custom-error-message')
             );
 
-        return this._ui.$errorsWrapper
+        var replaceErrorMessage = this._ui.$errorsWrapper
           .addClass('filled')
+          .attr('aria-hidden', 'false')
           .find('.parsley-custom-error-message')
           .html(this.options.errorMessage);
+
+        var addDescribedby = this._ui.$errorClassHandler
+          .attr('aria-describedby', this._ui.errorsWrapperId);
+
+        return replaceErrorMessage && addDescribedby;
       }
 
-      return this._ui.$errorsWrapper
+      var removeErrorMessage = this._ui.$errorsWrapper
         .removeClass('filled')
+        .attr('aria-hidden', 'true')
         .find('.parsley-custom-error-message')
         .remove();
+
+      var removeDescribedby = this._ui.$errorClassHandler
+        .removeAttr('aria-describedby');
+
+      return removeErrorMessage && removeDescribedby;
```

Comment:
This return statement makes no sense to me. Simplest change would be to first `removeAttr('aria-describedby')` then do as before, no?

Reply:
Bonjour Marc andré, 

I am not sure to understand. In the case there is no input assistance, it is necessary to remove aria-describedby. If we do not remove aria-describedby the technical assistance will try to read an empty ul.

Reply:
Please re-read carefully what I wrote. Nowhere do I propose not removing the `aria-describedby`.

Reply:
Hello Marc André,

Sorry this pull request is open for some time now.
Maybe it is possible to do more simple, but I am not a JS expert. The chunk of code you are highlighting just allow to remove the aria-describedby and the error message. If it is possible to make more simple do not hesitate to refactoring my code. If you need more explanation about the this accessibility changes I can explain more.